doc/nrf/gs_assistant.rst

```diff
@@ -9,11 +9,8 @@ Installing automatically
 
 `nRF Connect for Desktop`_, a cross-platform tool available for Windows, Linux, and macOS, provides different apps that simplify installing the |NCS|.
 
-.. note::
-   When installing the |NCS| through the |GSA| app, you will not be able to build on the command line using west.
-
 On Windows or macOS, use the :ref:`gs_app_tcm`.
-On Linux, use the :ref:`gs_app_gsa` or follow the steps found on the :ref:`gs_installing` page.
+On Linux, use the `Installing using Visual Studio Code`_ or follow the steps found on the :ref:`gs_installing` page.
```

Comment:
Do we need to mention about Visual Studio Code here as the user will be redirected to same gs_installing page from the Visual Studio Code. Thoughts?